README.md

```diff
@@ -1,44 +1,88 @@
-Guides
-======
+# Guides
 
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Guides for getting things done, programming well, and programming in style.
-
-* [Best Practices](./best-practices)
-* [Code Review](./code-review)
-* [Tech Stack](./stack)
-* [How to](./how-to)
-* [Protocol](./protocol)
-  * [Communication](./protocol/communication)
-  * [Accessibility](./protocol/accessibility)
-  * [Git](./protocol/git)
-  * [iOS](./protocol/ios)
-  * [Open Source](./protocol/open-source)
-  * [Product Review](./protocol/product-review)
-  * [Rails](./protocol/rails)
-  * [React](./protocol/react)
-* [Security](./security)
-* [Style](./style)
-* [Web Performance](./web-performance)
-* [Working Together](./working-together)
-
-High level guidelines:
+Guides for working together, getting things done, programming well, and programming in style.
+
+### High level guidelines
```

Comment:
I like moving this here.

Reply:
I also noticed we're going from a `h1` to a `h3`. Mind updating these to be `h2`s?